File: push_sale_mobile-master/lib/models/country.dart
class Country {
  final int id;
  final String name;
  final String code;
  Country({
    required this.id,
    required this.name,
    required this.code,
  });

  static Country fromMap(Map<String, dynamic> value) {
    return Country(
      id: value["id"],
      name: value["name"],
      code: value["code"],
    );
  }
}

File: push_sale-master/push_sale-master/app/Models/Country.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Laravel\Passport\HasApiTokens;

class Country extends Model
{
    use HasApiTokens,HasFactory;

    protected $table = "country";

    protected $fillable = [
        'name',
        'code',
    ];

    protected $hidden = ["created_at","updated_at"];


    public function Visit()
    {
        return $this->hasMany(VisitClient::class)->whereBetween('created_at', [date('Y-m-d 00:00:00'), date('Y-m-d 23:59:59')]);
    }
}
